sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/RoomsAsyncClient.java

```diff
@@ -339,20 +335,14 @@ Mono<AddOrUpdateParticipantsResult> addOrUpdateParticipants(String roomId, Itera
 
             Map<String, ParticipantProperties> participantMap = convertRoomParticipantsToMapForAddOrUpdate(participants);
 
-            ObjectMapper mapper = new ObjectMapper();
-
-            String updateRequest = mapper.writeValueAsString(new UpdateParticipantsRequest().setParticipants(participantMap));
-
+            String updateRequest = new UpdateParticipantsRequest().setParticipants(participantMap).toString();
```

Comment:
where the `toString` implementation comes from?

I doesn't seem like `JsonSerializable` or `UpdateParticipantsRequest` override the default implementation

Reply:
updated to use a StringWriter